frontend/src/scenes/userLogic.tsx

```diff
@@ -95,9 +95,10 @@ export const userLogic = kea<userLogicType<UserType, EventProperty, UserUpdateTy
                             posthog.reset()
                         }
 
-                        posthog.identify(user.distinct_id, {
-                            email: user.anonymize_data ? null : user.email,
```

Comment:
I think _part_ of the reason I wrote this this was was to make sure that if we accidentally set the email again it'll be overwritten. Does this not work as expected?

Reply:
This does not work at all. See https://github.com/PostHog/posthog/issues/1955

Reply:
Ah, I get your intent now. Updating!